bluez/audio/avdtp.c

```diff
@@ -2590,6 +2594,12 @@ static void avdtp_confirm_cb(GIOChannel *chan, gpointer data)
 			goto drop;
 		}
 		btd_device_add_uuid(dev->btd_dev, ADVANCED_AUDIO_UUID);
+		if (!main_opts.reverse_sdp && read_remote_class(&src, &dst, &class) == 0) {
+			if ((class & (1 << 21)) && (class & (1 << 18)))
+				btd_device_add_uuid(dev->btd_dev, A2DP_SINK_UUID);
+			if ((class & (1 << 21)) && (class & (1 << 19)))
```

Comment:
Aren't there symbolic constant for those bits? If not, this is fine I guess, it probably looks fairly obvious to someone intimately familiar with Bluetooth.